### PR TITLE
Update Essentials Connectivity API Docs

### PR DESCRIPTION
**Pull request by jfversluis**
### Description of Change

Moves the external comments to inline and minor improvements.

### Issues Fixed

Related to #3960

### CODE REVIEW

File: src/Essentials/src/Connectivity/Connectivity.shared.cs [hunk 1]
```diff
@@ -7,26 +7,52 @@
 
 namespace Microsoft.Maui.Networking
 {
+	/// <summary>
+	/// The Connectivity APIs lets you monitor for changes in the device's network conditions, check the current network access, and how it is currently connected.
```
Comment (hartez) [inline src/Essentials/src/Connectivity/Connectivity.shared.cs:11]:
```suggestion
	/// The Connectivity API lets you monitor for changes in the device's network conditions, check the current network access, and determine how it is currently connected.
```

File: src/Essentials/src/Connectivity/Connectivity.shared.enums.cs [hunk 2]
```diff
@@ -1,32 +1,44 @@
...
+
+		/// <summary>The mobile/cellular data connection.</summary>
 		Cellular = 2,
-		/// <include file="../../docs/Microsoft.Maui.Essentials/ConnectionProfile.xml" path="//Member[@MemberName='Ethernet']/Docs/*" />
+
+		/// <summary>The ethernet data connection.</summary>
 		Ethernet = 3,
-		/// <include file="../../docs/Microsoft.Maui.Essentials/ConnectionProfile.xml" path="//Member[@MemberName='WiFi']/Docs/*" />
+
+		/// <summary>The WiFi data connection.</summary>
```
Comment (hartez) [inline src/Essentials/src/Connectivity/Connectivity.shared.enums.cs:20]:
```suggestion
		/// <summary>The Wi-Fi data connection.</summary>
```

**Review by hartez — CHANGES_REQUESTED** [review]

**Review by hartez — APPROVED** [review]

File: src/Essentials/src/Connectivity/Connectivity.shared.enums.cs (hunk 2 above)
Comment (Santiagosaiz) [reply src/Essentials/src/Connectivity/Connectivity.shared.enums.cs]: @Santiagosaiz4inittowinit

**Pull request merged**